web_src/less/_dashboard.less

```diff
@@ -81,6 +81,12 @@
 
     .ui.dropdown {
       max-width: 100%;
+
+      @media @mediaSm {
+        > .menu:not(.hidden) {
```

Comment:
Is the `:not(.hidden)` selector necessary? Otherwise L-G-T-M

Reply:
IIRC it was to have selector priority.

Reply:
On my side, `> .menu { position: static; } ` works fine (ps: `static` seems more consistent than `inherit`)

```
      @media @mediaSm {
        > .menu {
          position: static;
        }
      }
```


![image](https://user-images.githubusercontent.com/2114189/181505798-d3b7db2e-a221-4262-9b4a-cb1f6a87ff85.png)